src/App.scss

```diff
@@ -1 +1,51 @@
-// styles go here
+iframe {
+  display: none !important;
+}
+
+.App {
+  width: 900px;
+  margin: 0 auto;
+  background: rgba(0, 0, 0, 0.1);
+  padding: 40px;
+  border-radius: 14px;
+  margin-top: 20px;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-around;
+  padding-top: 15px;
+  a {
+    font-size: 24px;
+    text-decoration: none;
+    color: #000;
+    font-family: sans-serif;
+  }
+}
+
+.homePage {
+  h1 {
```

Comment:
add class to h1, don't nest it inside the other element if it's not a block's element